test/e2e/framework/ginkgowrapper.go

```diff
@@ -374,12 +378,18 @@ func withFeature(name Feature) interface{} {
 // on the current stability level of the feature, to emulate historic
 // usage of those tags.
 //
-// In addition, [Feature:Alpha] resp. [Feature:Beta] get added to support
-// skipping a test with a dependency on an alpha or beta feature gate in
-// jobs which use the traditional \[Feature:.*\] skip regular expression.
+// For label filtering, Alpha resp. Beta get added to the Ginkgo labels.
```

Comment:
The changelog entry uses "ginkgo labels", should be "Ginkgo labels" as it is here.

Reply:
we prefer the capital? ok

Reply:
It's the official spelling of that name.